simulator/registry.go

```diff
@@ -59,6 +68,15 @@ func TypeName(item mo.Reference) string {
 	return reflect.TypeOf(item).Elem().Name()
 }
 
+// ValuePrefix returns the value name prefix of a given object
+func ValuePrefix(typeName string) string {
```

Comment:
No need to export this.  We could also change TypeName -> typeName while you're at it, no need to export that func either.